pkg/controller/machine/controller.go

```diff
@@ -229,14 +229,14 @@ func (r *ReconcileMachine) Reconcile(ctx context.Context, request reconcile.Requ
 			// we can loose instances, e.g. right after request to create one
 			// was sent and before a list of node addresses was set.
 			if len(m.Status.Addresses) > 0 || !isInvalidMachineConfigurationError(err) {
-				klog.Errorf("%v: failed to delete machine: %v", machineName, err)
+				klog.Errorf("%q: failed to delete machine: %q", machineName, err)
 				return delayIfRequeueAfterError(err)
 			}
 		}
 
 		instanceExists, err := r.actuator.Exists(ctx, m)
 		if err != nil {
-			klog.Errorf("%v: failed to check if machine exists: %v", machineName, err)
+			klog.Errorf("%q: failed to check if machine exists: %q", machineName, err.Error())
```

Comment:
Errors should remain no?
```suggestion
			klog.Errorf("%q: failed to check if machine exists: %v", machineName, err)
```